### PR TITLE
Add some documentation to run new-e2e installer tests locally

### PR DESCRIPTION
**Pull request by KevinFairise2**
### What does this PR do?

Give some documentation to run new-e2e installer tests locally

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided. Except if the `qa/skip-qa` label, with required either `qa/done` or `qa/no-code-change` labels, are applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Comment by GustavoCaso** [comment]
Something that we could consider is also checking the values of `osversion` and `platform` when running the tests and if they are empty, we could exit the tests and showing a help message showing how to run the tests

**Review by GustavoCaso — APPROVED** [review]

File: test/new-e2e/README.md [hunk 1]
```diff
@@ -21,3 +21,18 @@ The `agent-sandbox: test current file` can be used to launch test on a file with
...
+### Installer tests
+
+Installer tests are tests owned by Agent Platform, that mostly test the installation of the agent on different platforms with different methods. These tests are defined in `./tests/agent-platform`.
+The following are considered as installer tests:
+- `./tests/agent-platform/install-script`: Test the installation of the agent using the install script
+- `./tests/agent-platform/step-by-step`: Test the installation of the agent without the install script, following the manual installation described on the official documentation ([for example](https://app.datadoghq.com/account/settings/agent/latest?platform=debian))
+- `./tests/agent-platform/upgrade`: Test the upgrade of the agent using the install script
+
+All these tests can be executed locally with the following command:
+`aws-vault exec sso-agent-sandbox-account-admin -- go test <test folder path> -v -args -osversion '<comma separated os version list>' -platform '<debian/centos/suse/ubuntu>' -arch <x86_64/arm64>`
```
Comment (pducolin) [inline test/new-e2e/README.md:36]: **💭 thought**
Wonder if we should use the inv task instead, as described in internal docs. Not sure if it allows passing extra params to `gotestsum`

**Review by pducolin — APPROVED** [review]

**Review by pducolin — APPROVED** [review]

**Pull request merged**